toolchest_client/tools/api.py

```diff
@@ -44,35 +43,36 @@ def bowtie2(inputs, output_path=None, database_name="GRCh38_noalt_as", database_
     return output
 
 
-def cellranger_mkfastq(inputs, samplesheet_name, output_path=None, tool_args=""):
-    """Runs Cell Ranger's mkfastq command via Toolchest.
+def cellranger_count(inputs, transcriptome_name="GRCh38", output_path=None, tool_args=""):
+    """Runs Cell Ranger's count command via Toolchest.
 
     :param inputs: Path (client-side) to be passed in as input.
     :param output_path: (optional) Path (client-side) where the output file will be downloaded.
-    :param samplesheet_name: Name of sample sheet. Expected to exist inside of "inputs".
+    :param transcriptome_name: Name of transcriptome. Defaults to `GRCh38`.
```

Comment:
Huh, interesting. Technically the database for Bowtie and some others would be a `genome_name`. Up to you how you name it – I see the benefit to keeping this consistent with the Cell Ranger interface, but I also see a risk that we start making other params for other tools very specific when they're not needed.

Reply:
I would keep this consistent with the cellranger interface for now (with the limited number of tools we have), but I'm open to changing this down the road.

Reply:
I'd argue that keeping this consistent with the cellranger interface is analogous to maintaining capitalization to be consistent with the naming of the tools on the command line – it keeps consistency for those who are moving CLI -> Toolchest, but it decreases consistency within Toolchest.

I think it's important to think about who is actually going to be using this, and why. We don't have good data yet on who will be coming from CLI vs. using the tool for the first time in Toolchest / using multiple tools on Toolchest, so I can't really make a case – but worth keeping in mind that this comes across as an inconsistency within Toolchest.

Reply:
(I'm okay with not changing this)

Reply:
I'll change it to `database_name` to be in line with our other tools, then.